readme.md

```diff
@@ -35,12 +39,15 @@ $ npm install --global sparkly
 $ sparkly --help
 
   Usage
-    sparkly <number> ...
-    echo <number> ... | sparkly
+    sparkly [opts] <number> ...
+    echo [opts] <number> ... | sparkly
```

Comment:
Also added `[opts]` in square brackets to show that it is an optional argument to `sparkly`.